steps/oasys/cms-offender-details.ts

```diff
@@ -0,0 +1,6 @@
+import { Page, expect } from '@playwright/test'
+
+export const clickCreateOffenderButton = async (page: Page) => {
+    await page.click('#B8409510857074181')
```

Comment:
These IDs aren't very descriptive, and almost seem randomly generated... makes me worry they'll end up being flakey.

No need to change them for this PR, but if we hit problems with them in future we might want to switch to user-facing / text-based selectors to line up with Playwright best-practices: https://playwright.dev/docs/selectors#prioritize-user-facing-attributes.